src/LanguageServer/Impl/Indexing/IndexingFileParser.cs

```diff
@@ -17,8 +17,8 @@ class IndexingFileParser {
 
         public void ParseForIndex(string path) {
```

Comment:
Since class is already named `IndexingFileParser` just `Parse` is sufficient.  `IndexingFileParser.ParseForIndex` sounds a bit mouthful :-)